I-D-Profile-Negotiation.xml

```diff
@@ -324,7 +323,7 @@ application/ld+json; profile="https://www.w3.org/ns/activitystreams"
 					can be used to provide an identifier
 					for the specific representation of a resource
 					that is enclosed in the message body.
-					For example, when a User Agent issues the following request
+					For example, when a user agent issues the following request
```

Comment:
It was @larsgsvensson who initially had the _User Agent_ spelling.

Reply:
I don't really care if we write "user agent" or "User Agent" and guess that the reviewers at IETF will tell us if one is preferred above the other